### PR TITLE
Refactor/cmc token price

### PR DESCRIPTION
**Pull request by SidharthK2**
## Linked issues
closes https://github.com/EveripediaNetwork/issues/issues/2459

### CODE REVIEW

**Comment by Aliiiu** [comment]
The marketcap is different from the one on iq.dashboard

File: src/services/cmcTokenDetails/index.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  },
+  baseQuery: fetchBaseQuery({
+    baseUrl: '/api/cmc-token-details',
+  }),
+  refetchOnMountOrArgChange: 60,
+  refetchOnFocus: true,
+  endpoints: (builder) => ({
+    getCmcTokenData: builder.query<
+      {
+        [tokenName: string]: {
```
Comment (Aliiiu) [inline src/services/cmcTokenDetails/index.ts:20]: Create a type for this to make it cleaner

**Review by Aliiiu — CHANGES_REQUESTED** [review]

**Review by Softdev1 — APPROVED** [review]
image

**Review by Softdev1 — COMMENTED** [review]
Please ensure to fix the feedback above @SidharthK2

**Comment by Aliiiu** [comment]
Please update the branch before you merge

**Review by Aliiiu — APPROVED** [review]

**Pull request merged**